### PR TITLE
Add OpenStack vxlan to MTU chart

### PR DESCRIPTION
**Pull request by gyliu513**
## Description
A few sentences describing the overall goals of the pull request's commits.
Please include
- the type of fix - (e.g. bug fix, new feature, documentation)
- some details on _why_ this PR should be merged
- the details of the testing you've done on it (both manual and automated)
- which components are affected by this PR
- links to issues that this PR addresses #988 

## Todos
- [ ] Tests
- [X] Documentation
- [ ] Release note

## Release Note

```release-note
Update default MTU for Ip-in-Ip as 1430
```

### CODE REVIEW

**Comment by gyliu513** [comment]
Can anyone help review this PR? Thanks!

File: master/usage/configuration/mtu.md [hunk 1]
```diff
@@ -22,12 +22,13 @@ packet.
...
 | 1460 (GCE) | 1460 | 1440 | 1410 |
 | 9001 (AWS Jumbo) | 9001 | 8981 | 8951 |
+| 1450 (OpenStack VXLAN) | 1450 | 1430 | 1400 |
 
 #### Default MTU sizes
 
 The default MTU for workload interfaces is 1500, this is to match the most
 common network MTU size.  The default MTU for the IP-in-IP tunnel device
-is 1440 to match the value needed in GCE.
+is 1430 to match the value needed in OpenStack VXLAN.
```
Comment (tmjd) [inline master/usage/configuration/mtu.md]: Is there a PR up to change the default MTU that is configured for the IP-in-IP tunnel?  We probably don't want to update this line until that is merged.
If there is one or has already been merged could you add a link to it, I tried searching for one but couldn't find it.

Comment (gyliu513) [inline master/usage/configuration/mtu.md]: Not yet, @tmjd can you please show me more detail where is the default MTU that is configured for the IP-in-IP tunnel?

Comment (tmjd) [inline master/usage/configuration/mtu.md]: I think this is the location https://github.com/projectcalico/felix/blob/master/config/config_params.go#L139 you would need to update.

Comment (gyliu513) [inline master/usage/configuration/mtu.md]: Thanks @tmjd created a PR here https://github.com/projectcalico/felix/pull/1533 , can you help review as well? Thanks!

Comment (caseydavenport) [reply master/usage/configuration/mtu.md]: In order to get this PR merge, can we remove this diff and just include the change to the table above?

We can add this back once the felix PR has been agreed / merged.

Comment (gyliu513) [reply master/usage/configuration/mtu.md]: Done @caseydavenport

**Pull request merged**